model/patch/db.go

```diff
@@ -250,6 +250,13 @@ func MostRecentPatchByUserAndProject(user, project string) db.Q {
 	}).Sort([]string{"-" + CreateTimeKey}).Limit(1)
 }
 
+// MostRecentPatchByPR returns the latest patch for a pr number
+func MostRecentPatchByPR(prNumber int) db.Q {
+	return db.Query(bson.M{
```

Comment:
This should probably also check for the project name, because we can have conflicting PR numbers